src/tiledb/cloud/vcf/ingestion.py

```diff
@@ -842,8 +842,10 @@ def ingest_samples_dag(
     ingest_resources: Optional[Mapping[str, str]] = None,
     verbose: bool = False,
     trace_id: Optional[str] = None,
+    batch_mode: bool = True,
     access_credentials_name: Optional[str] = None,
-) -> dag.DAG:
+    compute: bool = True,
+) -> Tuple[dag.DAG, Sequence[str]]:
```

Comment:
Since this might return `None` in the first position, this should be `Optional[dag.DAG]`